src/SSVOpenHexagon/Components/CPlayer.cpp

```diff
@@ -278,7 +264,7 @@ void CPlayer::updateInput(HexagonGame& mHexagonGame, ssvu::FT mFT)
     const int movement{mHexagonGame.getInputMovement()};
 
     const float currentSpeed =
-        mHexagonGame.getInputFocused() ? focusSpeed : speed;
+        *playerSpeedMult * (mHexagonGame.getInputFocused() ? focusSpeed : speed);
```

Comment:
I think this would make more sense if it was reading the player speed multiplayer directly from `mHexagonGame`, e.g.:

```cpp
const float currentSpeed =
    mHexagonGame.getPlayerSpeedMult() * (mHexagonGame.getInputFocused() ? focusSpeed : speed);
```

That way we avoid the need of passing an extra pointer around to the player.